{RepositoryName}/src/{Namespace}.Web/package.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
remove "/src/{Namespace}.Web/package-lock.json" file

Reply:
But in [OrdersModule](https://github.com/VirtoCommerce/vc-module-order/tree/release/3.0.0/src/VirtoCommerce.OrdersModule.Web) repository "package-lock.json" file is exist.
Need more comments on why "package-lock.json" file should be removed.

Reply:
oh, an update: everyone SHOULD commit package-lock to source control.